cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -2593,6 +2593,127 @@ TEST_F(ScalarTemporalTest, TestCeilTemporalMultipleSinceGreaterUnit) {
   CheckScalarUnary(op, unit, times, unit, ceil_15_years, &round_to_15_years);
 }
 
+TEST_F(ScalarTemporalTest, TestCeilTemporalMultipleSinceGreaterUnitZoned) {
+  std::string op = "ceil_temporal";
+  RoundTemporalOptions round_to_15_nanoseconds =
```

Comment:
Are these options repeated around? Perhaps they should be factored in a test fixture class?

Reply:
Hm, yeah that might be a nice idea. Let me check.

Reply:
Moved to fixture. Names are not very inspired (`round_to_13_seconds` vs `round_to_13_seconds_2`).